src/Liquidator.sol

```diff
@@ -21,8 +22,12 @@ import {ITrustedCreditor} from "./interfaces/ITrustedCreditor.sol";
  * @dev contact: dev at arcadia.finance
  */
 contract Liquidator is Ownable {
-    uint256 public maxAuctionTime = 14_400; //4 hours in seconds
-    uint256 public startPriceMultiplier; //2 decimals
+    uint256 public startPriceMultiplier; // 2 decimals
```

Comment:
Can these 3 uint256's be packed?

Reply:
It can be packed, however they need to be read quite frequently so read can be higher cost

Reply:
reading & writing packed slots are much cheaper as compared to reading/writing separate slots